sdk/core/core-util/src/datetime.ts

```diff
@@ -0,0 +1,32 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+export type OffsetMillisecondsPrecision = 0 | 1 | 2 | 3;
+
+export function toOffsetDateTime(date: Date, precision: OffsetMillisecondsPrecision = 0): string {
```

Comment:
add an option as the precision of milliseconds we want to keep. by default it's 0 now, but let me know if there;s other opppinion.